packages/backend/src/graphql/project/project.ts

```diff
@@ -28,11 +28,32 @@ export const Project = builder.prismaObject('Project', {
         orderBy: { title: 'asc' },
       }),
     }),
-    members: t.field({
+    members: t.prismaField({
       description: 'List of users that are member of the project',
-      select: { projectMemberships: { select: { user: true } } },
-      type: [User],
-      resolve: (project) => project.projectMemberships.map((projectMembership) => projectMembership.user),
+      select: { id: true },
+      type: ['User'],
+      args: {
+        includingUsersWhoBookedWorkHours: t.arg.boolean({
```

Comment:
maybe it would be better not to user project members for the report but using a different graphql object for the reports.

Reply:
or maybe more close to your solution
```suggestion
        includePastMembers: t.arg.boolean({
```